docker-compose.services.yml

```diff
@@ -2,44 +2,24 @@ version: '3.9'
 
 networks:
   online-store-network:
-    name: online-store-network
+    name: '${DOCKER_COMPOSE_NETWORK}'
 
 services:
   postgresdb:
     image: 'postgres:13.11-bullseye'
     container_name: online-store-postgresdb
     environment:
-      - POSTGRES_USER=postgres
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_DB=testdb
+      - POSTGRES_USER=${DATASOURCE_USERNAME}
+      - POSTGRES_PASSWORD=${DATASOURCE_PASSWORD}
+      - POSTGRES_DB=${APP_SERVER_PORT}
```

Comment:
If i am not wrong - here should be the DATASOURCE_PORT?

Reply:
Fixed